src/components/graph-view.js

```diff
@@ -723,8 +718,8 @@ class GraphView extends Component {
     let d3Node = d3.select(domNode);
     let title = d.title;
 
-    let titleText = title.length < maxTitleChars ? title : 
-      `${title.substring(0, maxTitleChars)}...`;
+    let titleText = title.length <= this.props.maxTitleChars ? title :
```

Comment:
changing `<` to `<=` since 9-char titles are getting "..." appended when it isn't necessary